tsconfig.json

```diff
@@ -16,10 +16,11 @@
     "strictNullChecks": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "noImplicitAny": true,
     "noImplicitThis": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true
+    "noUnusedParameters": true,
+    "allowSyntheticDefaultImports": true,
+    "allowJs": true
```

Comment:
Why are these changes needed?

Reply:
we just need to remove the **noImplicityAny** to be available to import the defaultMetadataStorage from the new version of class-transformer, otherwise would dispatch an error, the last 2 lines, are not necessary, reverted.

Reply:
Is there any way to use an explicit `as any` coercion and keep `"noImplicitAny": true`?